osmaxx-py/excerptconverter/dummyexcerptconverter/dummy_excerpt_converter.py

```diff
@@ -52,7 +54,7 @@ def export_options():
         }
 
     @staticmethod
-    def create_output_files(execution_configuration, extraction_order, supported_export_formats):
+    def create_output_files(execution_configuration, extraction_order, supported_export_formats, converter_helper):
```

Comment:
While it might be good to pass `converter_helper`, so that it can be replaced for isolated tests, we should probably have a default value here, so that production callers don't have to explicitly pass it.